daemon/export_api_themes_test.go

```diff
@@ -0,0 +1,37 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package daemon
+
+type (
+	ThemeStatus         = themeStatus
+	ThemeStatusResponse = themeStatusResponse
+)
+
+var (
+	ThemesCmd                    = themesCmd
```

Comment:
it woudl be nice not to need to export this, I think changing installedThemes to take an Overlord would help?

Reply:
I've updated installedThemes to take an Overlord as the argument.  The other user is themeStatusAndCandidateSnaps(), which is a bit trickier, since it is calling `getStore(*Command)`.

That helper only needs the Overlord too, so could be updated.  But it is used in about 10 other places.  Maybe this would be better to handle in a simple follow-up PR?

Reply:
I probably need to touch that helper myself in my ongoing cleanups,  maybe best to leave a

// XXX it would be nice to not export this

here for now